functions/test/helloworld/storage_generic_test.rb

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "helper"
+require "date"
+
+describe "functions_helloworld_storage" do
+  include FunctionsFramework::Testing
+
+  let(:source) { "//storage.googleapis.com/projects/sample-project/buckets/sample-bucket/objects/ruby-rocks.rb" }
+  let(:type) { "google.storage.object.change.v1" }
```

Comment:
Can you use `google.cloud.storage.object.v1.finalized` instead? The type mapping is changing (and yes, I do have to update the other samples accordingly...)

Reply:
Done.